DATA_PREPARATION.md

```diff
@@ -70,6 +70,7 @@ After everything is done, remember to run the commands below to do a final check
   - `vision_check_dataset stanford_cars -c train_coco.json -f ./ -t classification_multiclass`
   - `vision_check_dataset stanford_cars -c test_coco.json -f ./ -t classification_multiclass`
 
+Support for `key_value_pair` dataset is coming soon.
```

Comment:
It is probably needed, if you want ppl to start contributing datasets of it. It could be a hassle and bug-prone without this.